catalog/nsili/catalog-nsili-source/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -36,6 +36,15 @@
         <AD description="Poll Interval to Check if the Source is available (in minutes - minimum 1)."
             name="Poll Interval" id="pollInterval"
             required="true" type="Integer" default="5"/>
+
+        <AD description="User ID for CORBA Service Access"
```

Comment:
I'm wondering if we shouldn't use a description like `NSILI Access Criteria Username` or something instead of CORBA, since the AccessCriteria stuff is a part of GIAS?